engine/eConstr.mli

```diff
@@ -46,6 +46,14 @@ sig
   (** Returns the view into the current sort. Note that the kind of a variable
       may change if the unification state of the evar map changes. *)
 
+  val equal : Evd.evar_map -> t -> t -> bool
```

Comment:
Might be worth having a couple more apis like `ESorts.set` `ESorts.is_sprop`

Reply:
I thought about that but it's unclear if we can really leverage them in the long run. The parts that mess with universes currently do it in unsafe ways.

Reply:
I don't know what you mean by that. 

I just think `ESorts.is_prop sigma s` would be nicer to use than `Sorts.is_prop (ESorts.kind sigma s)`

Reply:
OK, OK, I will add some.